app/scripts/stores/post.js

```diff
@@ -1,19 +1,69 @@
 var Reflux = require('reflux');
-
+var UUID = require('uuid');
 var PostActions = require('../actions/post');
 
 // Deals with App State Machine state
 var PostStore = Reflux.createStore({
 
+    posts: {},
+
     init: function() {
 
+        // fetches the list of most recent posts
+        this.posts = this.getPosts();
+
         // Listeners
-        this.listenTo(PostOptions.addPost, this.addPost);
+        this.listenTo(PostActions.newPost, this.addPost);
+    },
+
+    getPosts: function (query) {
+      // TODO: actually fetch posts, rather than used fake data
+      return {
```

Comment:
@lanre-ade @mraypold @jcairo here's the structure of what a get post could potentially look like.